ui/app/src/modules/Search/FilterSearchDropdown.tsx

```diff
@@ -828,9 +831,9 @@ export default function FilterSearchDropdown(props: FilterSearchDropdownProps) {
         }}
       >
         <List classes={{ padding: classes.listPadding }}>
-          <div>
+          <>
```

Comment:
What’s the fragment doing here? Same goes for the one next too.

Reply:
nothing